README.md

```diff
@@ -8,7 +8,12 @@ dsd stands for "DNA sequence designer". It is a Python library that enables one
 ## Installation
 dsd requires Python version 3.6 or higher. Currently, it cannot be installed using pip (see [issue #12](https://github.com/UC-Davis-molecular-computing/dsd/issues/12)). 
 
-dsd uses [NUPACK](http://www.nupack.org/downloads) and [ViennaRNA](https://www.tbi.univie.ac.at/RNA/#download). While it is technically possible to use dsd without them, most of the pre-packaged constraints require them. ViennaRNA is fairly straightforward to install on any system. Installing NUPACK requires compiling C code from source, and it is not straightforward to install on Windows (for example, using Cygwin or MinGW). To use NUPACK on Windows, you should use [Windows Subsystem for Linux](https://docs.microsoft.com/en-us/windows/wsl/install-win10), which essentially installs a command-line-only Linux inside of your Windows system.
+dsd uses [NUPACK](http://www.nupack.org/downloads) and [ViennaRNA](https://www.tbi.univie.ac.at/RNA/#download). While it is technically possible to use dsd without them, most of the pre-packaged constraints require them. ViennaRNA is fairly straightforward to install on any system.
```

Comment:
Let's add some description of how someone can test that they installed NUPACK 4 successfully, to be added to the current step 5 of the installation instructions: 

![image](https://user-images.githubusercontent.com/19274365/104453242-f755a400-5558-11eb-9b61-cbd98fee83b9.png)

Since it said above that they can install NUPACK 3 *or* 4, make sure to specify in these instructions that which test they should do depends on which they installed. Should they avoid installing both? I could imagine both have an executable named `pfunc`, and the current technique of calling it from the command line while assuming it is on the PATH would not work, since one of them would be found on the path first. Make sure to spell out these details in the installation instructions. (Or we can devise new installation instructions, where we allow the user to point us to the correct executable, for instance through a new environment variable, or a parameter to the relevant Python functions.)

Reply:
Installing both should be fine and does not interfere with one another. NUPACK 3 installs a `pfunc` executable but NUPACK 4 installs a python package via `pip`. Users can test out NUPACK 4 installation by just running
``` python
>>> dv.pfunc4('GCGCGCGCGC')
-1.9079766874655928
```

Or perhaps pass in parameters to make the result closer  to the result of testing NUPACK 3:
```python
>>> val = dv.pfunc4('GCGCGCGCGC', sodium=1.0, magnesium=0.0)
-2.160513520226214
```

I'm not sure why the latter call does not exactly match the result of NUPACK 3 (`-2.06056146`). Based on the source code for NUPACK 4, the default `sodium` and `magnesium` should be 1 and 0 respectively and the default `temperature` is 37. There is either some other parameter that is being set differently or the numerical computation changed from NUPACK 3 to NUPACK 4.

Reply:
Can you add

> Users can test out NUPACK 4 installation by just running
> 
> ```python
> >>> dv.pfunc4('GCGCGCGCGC')
> -1.9079766874655928
> ```

Can you add this to the README? (and emphasize what each is testing, i.e., that the existing `dv.pfunc('GCGC')` tests whether NUPACK 3 is installed and will fail if they only installed NUPACK 4.